test/reducer.test.js

```diff
@@ -38,22 +58,22 @@ const prevActionToRetry1WithDifferentPayload = {
 
 describe('unknown action type', () => {
   it('returns prevState on initialization', () => {
-    expect(reducer(undefined, { type: 'ACTION_I_DONT_CARE' })).toEqual(
+    expect(createReducer()(undefined, { type: 'ACTION_I_DONT_CARE' })).toEqual(
```

Comment:
Since `createReducer()` is sort of repetitive in all tests, let's factor that out into a variable we can reuse, like

`const networkReducer = createReducer()`